po/cs.po

```diff
@@ -0,0 +1,268 @@
+# Czech translations for com.github.elfenware.badger package.
+# Copyright (C) 2021 THE com.github.elfenware.badger'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the com.github.elfenware.badger package.
+# Vojtěch Perník <translations@pervoj.cz>, 2021.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: com.github.elfenware.badger\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2019-10-24 01:00+0530\n"
+"PO-Revision-Date: 2021-08-09 17:52+0200\n"
+"Last-Translator: Vojtěch Perník <translations@pervoj.cz>\n"
+"Language-Team: Czech <gnome-cs-list@gnome.org>\n"
```

Comment:
What is this email about? Is it okay to include the GNOME translation list?

Reply:
I'm also a member of the mailing list, so I don't think that should be a problem.